app/konnect/analytics/reference.md

```diff
@@ -0,0 +1,107 @@
+Request Size | The size of the request payload received from the client, in bytes. Users can select between different percentiles (p99, p95, and p50). For example, a 99th percentile request size of 100 bytes means that the payload size for every 1 in 100 requests was at least 100 bytes.
+Response Size | The size of the response payload returned to the client, in bytes. Users can select between different percentiles (p99, p95, and p50). For example, a 99th percentile response size of 100 bytes means that the payload size for every 1 in 100 response back to the original caller was at least 100 bytes.
+
+{:.note}
+> **Note:** {{site.konnect_saas}} Analytics uses percentiles to enable you to understand the real performance characteristics of your APIs. Percentiles depict a more accurate picture of what most end users experience using your API instead of hiding critical experiences by displaying averages.
+
+## Grouping and filtering
+
+When selecting a category to filter or group by, you have the following options:
+
+Category | Description
+---------|------------
+None | Aggregate all of the data in the organization without any grouping.
+Service | Group or filter the data by {{site.konnect_short_name}} service.
+Service Version | Group or filter the data by {{site.konnect_short_name}} service version. <br><br>If you select multiple service versions in a report, the report shows the sum of requests for all selected versions broken down by service. It **does not** show data points for individual service versions.
+Route | Group or filter the data by route. Route names are composed of elements referencing their related service, service version, and runtime group. See [Route entity format](#route-entity-format) for details.
+Application | Group or filter the data by application.
+Status Code | Group or filter the data by individual response status code. Displays visualizations of cluster-wide status code classes (1xx, 2xx, 3xx,
+  4xx, 5xx). The Status Codes view contains the counts of status code classes
+  graphed over time, as well as the ratio of code classes to total requests.
+Status Code (grouped) | Group or filter the data by response status code category: 1XX, 2XX, 3XX, 4XX, and 5XX.
+
+### Route entity format
+
+In custom reports, the route entity name is composed of the following elements:
+
+```
+KONNECT_SERVICE_NAME.VERSION.ROUTE_NAME|FIRST_FIVE_UUID_CHARS (RUNTIME GROUP)
+```
+
+For example, for a route entity named `example_service.v1.example_route` with the badge `default`:
+* `example_service` is the {{site.konnect_short_name}} service name
+* `v1` is the service version
+* `example_route` is the route name
+* `default` is the runtime group name
+
+Or, if your route doesn't have a name, it might look like this:
+`example_service.v1.DA58B`
+
+Where `DA58B` are the first five characters of its UUID.
+
+### Service version entity format
```

Comment:
@sichvoge Is this what you had in mind?

Reply:
Yes. How about we add some explanation to it as well. Like: 

> "Service Version" name is not unique across an organization. Hence, for you to better identify the entity you need, the service version entity name is composed of the following elements:

Or something like that :D